aiida_testing/export_cache/_fixtures.py

```diff
@@ -345,6 +345,8 @@ def _run_with_cache( # type: ignore
 
         # check existence
         full_import_path = pathlib.Path(data_dir) / (name + '.tar.gz')
+        # make sure the path is absolute (this is needed by export_cache)
+        full_import_path  = full_import_path.absolute()
```

Comment:
if this is not an absolute path `export_cache` fails because there this is checked

Reply:
I think we should _check_ that it is an absolute path instead of just making it absolute. Otherwise the test will depend on which directory they are run from. 

As much as possible, e.g. running `pytest` from the root directory should do the same as e.g. `cd tests; pytest`.

Reply:
oh yes, that slipped my mind. so how about we make this a check only and get rid of the default value. Then the users would be required to set the paths to executables, output directory etc. themselves and we wouldn't be hiding this in the fixture.
I guess this would be cleaner.